source/fundamentals/linq.txt

```diff
@@ -0,0 +1,541 @@
+.. _csharp-linq:
+
+====
+LINQ
+====
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Overview
+--------
+
+In this guide you can learn how to use
+`LINQ <https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/linq/?redirectedfrom=MSDN>`__
+with the {+driver-long+}. LINQ allows you to construct queries against
+strongly typed collections of objects by using language keywords and operators.
+The {+driver-short+} automatically translates LINQ queries into
+:manual:`aggregation operations </aggregation>`.
+
+The examples in this guide use the ``restaurants`` collection
+in the ``sample_restaurants`` database provided in the :atlas:`Atlas sample datasets </sample-data>`.
+To learn how to create a free MongoDB Atlas cluster and load the sample datasets,
+see the :ref:`<csharp-quickstart>`.
+
+The following ``Restaurant`` class models the documents in this collection:
+
+.. literalinclude:: /includes/fundamentals/code-examples/linq.cs
+   :language: csharp
+   :dedent:
+   :start-after: start-restaurant-model
+   :end-before: end-restaurant-model
+
+.. _csharp-linq-queryable:
+
+Make A Collection Queryable
+---------------------------
+
+To use LINQ to query your collection you must first make the collection
+queryable. To make your collection queryable, use the ``AsQueryable`` method
+as follows:
+
+.. code-block:: csharp
+   :emphasize-lines: 2
+
+    _restaurantsCollection = db.GetCollection<Restaurant>("restaurants");
+    var queryableCollection = _restaurantsCollection.AsQueryable();
+
+Once the collection is queryable, you can compose a query using 
+**query comprehension syntax** or **method syntax**.
```

Comment:
Issue: Since some of the stages don't support query syntax, can we mention that here? 

Suggestion: Create a Note box mentioning that. Or change this paragraph to something like:

```suggestion
Once the collection is queryable, you can compose a query using **method syntax**. Some pipeline stages also support **query comprehension syntax**, which resembles SQL syntax.
```

Reply:
Changed the paragraph to your suggestion